### PR TITLE
[#12334] Instructor courses page: fix copy instructors from other courses modal checkbox

### LINKED ISSUE #12334: Instructor courses page: fix copy instructors from other courses modal checkbox
Path: Instructor pages -> Courses tab -> Edit course details -> Copy instructors from other courses

The modal that appears has a checkbox to select instructors to copy over. However, there's no indication that it has been checked:
copy instructors modal

### PR DESCRIPTION
**Pull request by judith-n**
Fixes #12334

### CODE REVIEW

**Review by cedricongjh — CHANGES_REQUESTED** [review]
Hi @judith-n, thank you for your pull request!

Functionally, the checkbox now works:
image

Let's also take the chance to standardise it's appearance, as it is different from the other checkboxes throughout the application:
image
image

This is most likely due to the `form-control` class of the input, which sets the checkbox height and width. Removing the `form-control` class from the input that should result in the checkbox looking the same as the rest.

Do make the change and

File: src/web/app/pages-instructor/instructor-course-edit-page/copy-instructors-from-other-courses-modal/copy-instructors-from-other-courses-modal.component.html [hunk 1]
```diff
@@ -79,7 +79,7 @@ <h4 class="modal-title">Copy Instructors</h4>
                           <input disabled type="checkbox" class="form-control no-shadow">
                         </div>
                         <div *ngIf="instructorCandidate.instructor.role !== InstructorPermissionRole.INSTRUCTOR_PERMISSION_ROLE_CUSTOM">
-                          <input id="enabled-checkbox" [disabled]="isCopyingSelectedInstructors" type="checkbox" class="form-control no-shadow" [(ngModel)]="instructorCandidate.isSelected">
+                          <input id="enabled-checkbox" [disabled]="isCopyingSelectedInstructors" type="checkbox" class="form-control form-check-input no-shadow" [(ngModel)]="instructorCandidate.isSelected">
```
Comment (cedricongjh) [inline src/web/app/pages-instructor/instructor-course-edit-page/copy-instructors-from-other-courses-modal/copy-instructors-from-other-courses-modal.component.html:82]:
```suggestion
                          <input id="enabled-checkbox" [disabled]="isCopyingSelectedInstructors" type="checkbox" class="form-check-input no-shadow" [(ngModel)]="instructorCandidate.isSelected">
```

**Comment by judith-n** [comment]
@cedricongjh thank you so much! :)

**Review by cedricongjh — APPROVED** [review]
LGTM, thank you for your contribution to TEAMMATES!

**Review by zhaojj2209 — APPROVED** [review]
LGTM!

**Pull request merged**